src/Domain/Creators/FrustalCloak.hpp

```diff
@@ -52,25 +52,23 @@ class FrustalCloak : public DomainCreator<3> {
     using type = bool;
     static constexpr Options::String help = {
         "Use equiangular instead of equidistant coordinates."};
-    static constexpr type default_value() noexcept { return false; }
   };
 
   struct ProjectionFactor {
     using type = double;
     static constexpr Options::String help = {"Grid compression factor."};
-    static constexpr type default_value() noexcept { return 1.0; }
   };
 
   struct LengthInnerCube {
     using type = double;
     static constexpr Options::String help = {"Side length of each inner cube."};
-    static constexpr type default_value() noexcept { return 0.0; }
```

Comment:
To answer the earlier question (if 0.0 is a valid value), if I remember correctly some default values where chosen explicitly to be nonsense values to force the user to set them explicitly after the run failed fast. It was better than the alternative where no value would be set and a (probably) random value would be set for the side length, making the error harder to detect than if it were just zero.